Out of the box it can't find the shared-lib

Error: src/app/components/mutiples/mutiples.component.ts:4:32 - error TS2307: Cannot find module 'shared-lib' or its corresponding type declarations.

import { WindowApiConst } from 'shared-lib';

--- a/.husky/_/husky.sh
+++ b/.husky/_/husky.sh
@@ -23,8 +23,7 @@ if [ -z "$husky_skip_init" ]; then
 
   if [ $exitCode != 0 ]; then
     echo "husky - $hook_name hook exited with code $exitCode (error)"
-    exit $exitCode
   fi
 
-  exit 0
+  exit $exitCode
 fi

--- a/workspaces/shared-lib/.dist/apis/window-api-consts.d.ts
+++ b/workspaces/shared-lib/.dist/apis/window-api-consts.d.ts
@@ -1,0 +1,10 @@
+export declare class WindowApiConst {
+	/** Channel used by the renderer process to send data to the main process */
+	static readonly MULTIPLES_INPUT = 'getMultiplesInput';
+	/** Channel used by the renderer process to receive data from the main process */
+	static readonly MULTIPLES_OUTPUT = 'getMultiplesOutput';
+	/** Whitelist of the safe channels to use when sending data to the main process */
+	static readonly SENDING_SAFE_CHANNELS: string[];
+	/** Whitelist of the safe channels to use when receiving data from the main process */
+	static readonly RECEIVING_SAFE_CHANNELS: string[];
+}

--- a/workspaces/shared-lib/.dist/apis/window-api-consts.js
+++ b/workspaces/shared-lib/.dist/apis/window-api-consts.js
@@ -1,0 +1,10 @@
+export class WindowApiConst {}
+/** Channel used by the renderer process to send data to the main process */
+WindowApiConst.MULTIPLES_INPUT = 'getMultiplesInput';
+/** Channel used by the renderer process to receive data from the main process */
+WindowApiConst.MULTIPLES_OUTPUT = 'getMultiplesOutput';
+/** Whitelist of the safe channels to use when sending data to the main process */
+WindowApiConst.SENDING_SAFE_CHANNELS = [WindowApiConst.MULTIPLES_INPUT];
+/** Whitelist of the safe channels to use when receiving data from the main process */
+WindowApiConst.RECEIVING_SAFE_CHANNELS = [WindowApiConst.MULTIPLES_OUTPUT];
+//# sourceMappingURL=window-api-consts.js.map

--- a/workspaces/shared-lib/.dist/apis/window-api.d.ts
+++ b/workspaces/shared-lib/.dist/apis/window-api.d.ts
@@ -1,0 +1,14 @@
+export interface WindowApi {
+	/**
+	 * This method is used by the renderer process to receive data from the main process
+	 * @param channel used by the renderer to receive data and by the main to send them
+	 * @param func the callback function to execute when data are available
+	 */
+	receive<Out>(channel: string, func: (output: Out) => void): void;
+	/**
+	 * This method is used by the renderer process to send data to the main process
+	 * @param channel used by the renderer to send data and by the main to receive them
+	 * @param data the data sent by the renderer process to the main process
+	 */
+	send<In>(channel: string, input: In): void;
+}

--- a/workspaces/shared-lib/.dist/apis/window-api.js
+++ b/workspaces/shared-lib/.dist/apis/window-api.js
@@ -1,0 +1,2 @@
+export {};
+//# sourceMappingURL=window-api.js.map

--- a/workspaces/shared-lib/.dist/index.d.ts
+++ b/workspaces/shared-lib/.dist/index.d.ts
@@ -1,0 +1,9 @@
+import { WindowApi } from './apis/window-api';
+export * from './apis/window-api';
+export * from './apis/window-api-consts';
+export * from './models/config/app-config';
+declare global {
+	interface Window {
+		api: WindowApi;
+	}
+}

--- a/workspaces/shared-lib/.dist/index.js
+++ b/workspaces/shared-lib/.dist/index.js
@@ -1,0 +1,4 @@
+export * from './apis/window-api';
+export * from './apis/window-api-consts';
+export * from './models/config/app-config';
+//# sourceMappingURL=index.js.map

--- a/workspaces/shared-lib/.dist/models/config/app-config.d.ts
+++ b/workspaces/shared-lib/.dist/models/config/app-config.d.ts
@@ -1,0 +1,18 @@
+export interface AppConfig {
+	/** The configuration identifier */
+	configId: string;
+	/** The main logger output file path */
+	mainLogFile: string;
+	/** The main logger output level */
+	mainLogLevel: string;
+	/** Tells if we should try to load app icon */
+	isIconAvailable: boolean;
+	/** Tells if `nodeIntegration` webPreference is enabled */
+	isNodeIntegration: boolean;
+	/** Tells if  `contextIsolation` and `worldSafeExecuteJavaScript` webPreferences are enabled */
+	isContextIsolation: boolean;
+	/** Tells if `isEnableRemoteModule` webPreference is enabled */
+	isEnableRemoteModule: boolean;
+	/** Tells if we should open dev tools */
+	isOpenDevTools: boolean;
+}

--- a/workspaces/shared-lib/.dist/models/config/app-config.js
+++ b/workspaces/shared-lib/.dist/models/config/app-config.js
@@ -1,0 +1,2 @@
+export {};
+//# sourceMappingURL=app-config.js.map

--- a/workspaces/shared-lib/.gitignore
+++ b/workspaces/shared-lib/.gitignore
@@ -1,2 +1,0 @@
-# Generated JS files
-/.dist

